.github/workflows/tests.yml

```diff
@@ -43,6 +45,8 @@ jobs:
             php: 7.2
           - laravel: 8.*
             php: 7.2
+          - laravel: 9.*
+            php: 8.0
```

Comment:
You need to exclude versions of PHP that Laravel 9 does not support.

Reply:
Tnx. Fixed.